packages/mobile/locales/en-US/onboarding.json

```diff
@@ -20,7 +20,7 @@
     "create": "Create a PIN",
     "verify": "Enter PIN again",
     "pinsDontMatch": "The PINs didn't match",
-    "invalidPin": "Invalid PIN"
+    "invalidPin": "Please use a stronger PIN"
```

Comment:
This is only shown if the pin was blacklisted, so we can assume the reason is weak pin